source/_includes/installation/operating_system.md

```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
     More details can be found about the command can be found [here](https://www.virtualbox.org/manual/ch08.html#vboxmanage-storageattach).
 
     </div>
+
+- title: Unraid
```

Comment:
This has been scoped by an architectural decision, see:

<https://github.com/home-assistant/architecture/blob/master/adr/0015-home-assistant-os.md>

We don't want to support/add more than decided in there. If we want to extend this list, we need to discuss this in an architectural discussion and approve this first, before we add this to the documentation.

Feel free to open up an architectural proposal in our architecture repo.

../Frenck

Reply:
I was reading there and decided to add this to the docs based on:

> We will provide documentation for the following systems build on top of these technologies:
>  Proxmox (KVM/QEMU based)
>  Unraid (KVM/QEMU based)
>  ...

What am I missing?

Reply:
You? Nothing. Me? A lot apparently 🙈

Reply:
Hehe… Super! 👍🏻
Is it too much to ask to get the hacktoberfest label back also? 🫣